spec/classes/mod/lbmethod_bybusyness_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe 'apache::mod::lbmethod_bybusyness', type: :class do
+  context 'on a Debian OS' do
+    include_examples 'Debian 11'
+
+    context 'with Apache version >= 2.4' do
```

Comment:
You start running into api change with this since `apache::apache_version` has to be set to `apache::mod::lbmethod_heartbeat` currently If you remove the necessity for that parameter and in fact the parameter completely then that's the api change.

While that's probably a good thing a separate patch I'd say.

Reply:
Looks like I missed that in cedd45b63be89ea54bd2a596e6cd3a3f60d4faf8. We already dropped it from `apache::balancer` and everywhere else. Is it really breaking the API? The whole `$apache::apache_version` parameter doesn't exist anymore, which is probably why it's broken by default.

I've opened https://github.com/puppetlabs/puppetlabs-apache/pull/2552 to resolve it in a compatible way. I'd prefer to base this PR on that.

Reply:
> I'd prefer to base this PR on that.

Not the other way around. 
* Add tests to confirm current behaviours.
* Change behaviour using tests to confirm that change of behaviour.